src/lib/lss.ts

```diff
@@ -239,3 +246,144 @@ export async function createSplitsXml(config: Config): Promise<string> {
     }
   );
 }
+
+export function importSplitsXml(str: string): Config {
+  // xml parse
+  const parser = new DOMParser();
+  const xmlDoc = parser.parseFromString(str, "text/xml");
+  // GameName, CategoryName -> gameName, categoryName
+  const gameName =
+    xmlDoc.getElementsByTagName("GameName")[0].textContent?.trim() || "";
+  const categoryName =
+    xmlDoc.getElementsByTagName("CategoryName")[0].textContent?.trim() || "";
+  // Metadata Variables -> variables
+  const variablesVariables = xmlDoc
+    .getElementsByTagName("Variables")[0]
+    .getElementsByTagName("Variable");
+  let variables: Record<string, string> | undefined;
+  if (variablesVariables && variablesVariables.length > 0) {
+    // vs is mutated as an object, but not as a variable
+    const vs: Record<string, string> = {};
```

Comment:
we have `variables`, `variablesVariables`, and `vs` in scope here. these are a confusing set of names

`vs` seems like actually a useless piece of indirection. this loop can just mutate `variables` directly, no?

then `variablesVariables` could be renamed to something like `xmlDocVariables`

Reply:
Renaming `variablesVariables` to `xmlDocVariables` is a good idea.

The reason for `variables` vs `vs` is that `variables` can be undefined, but `vs` is always a Record so I can assign to it without checking for undefined.